modules/cli/src/main/java/org/apache/ignite/cli/commands/connect/ConnectCommand.java

```diff
@@ -44,7 +46,7 @@ public class ConnectCommand extends BaseCommand implements Runnable {
     @Override
     public void run() {
         CallExecutionPipeline.builder(connectCall)
-                .inputProvider(() -> new ConnectCallInput(nodeUrl))
+                .inputProvider(() -> new ConnectCallInput(nodeUrl.toString()))
```

Comment:
Shouldn't we handle the `nodeUrl==null` case here? Doesn't `NodeUrlMixin` suit well here?

Reply:
In this command `nodeUrl` is a parameter, not an option, and it can't be null - it will taken from either the parameter or from the config via the `descriptionKey`, and will throw an error in case the config doesn't have this parameter defined.
I added a test which shows that.

Reply:
Oh, I see, thanks.